### PR TITLE
SOC-1476 | Move discussion translations to discussion.json file

### PR DESCRIPTION
**Pull request by Bloodmoon24**
https://wikia-inc.atlassian.net/browse/SOC-1476 
@kvas-damian  @bkoval  please CR

### CODE REVIEW

File: server/views/_helpers/translations.js [hunk 1]
```diff
@@ -6,26 +6,28 @@ module.exports = function (language, opts) {
...
 		defaultLanguage = 'en',
 		foundLanguage = '',
-		namespace = opts.hash.ns || 'main',
+		namespace = opts.hash.ns,
 		wrapper = {};
 
-	[language, fallbackLanguage, defaultLanguage].some(function (lang) {
-		foundLanguage = lang;
-		translationPath = '../../../front/locales/' + lang + '/' + namespace + '.json';
+		namespace.forEach(function(ns){
```
Comment (kvas-damian) [inline server/views/_helpers/translations.js]: Please fix indentation

File: server/views/_helpers/translations.js [hunk 2]
```diff
@@ -6,26 +6,28 @@ module.exports = function (language, opts) {
 		fallbackLanguage = language.split('-')[0],
 		defaultLanguage = 'en',
 		foundLanguage = '',
-		namespace = opts.hash.ns || 'main',
+		namespace = opts.hash.ns,
```
Comment (kvas-damian) [inline server/views/_helpers/translations.js]: I think that we should still use default value here. What do you think about ```['main']``` ?

File: server/views/_helpers/translations.js [hunk 3]
```diff
@@ -6,26 +6,28 @@ module.exports = function (language, opts) {
...
+				} catch (exception) {
+					Logger.error({
+						lang: lang,
+						namespace: ns,
+						path: translationPath,
+						error: exception.message
+					}, 'Translation not found');
+				}
+			});
+			if(!wrapper.hasOwnProperty(foundLanguage)) wrapper[foundLanguage]= {};
```
Comment (kvas-damian) [inline server/views/_helpers/translations.js]: Please follow guidelines: https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#bad-examples

File: server/views/_layouts/ember-main.hbs [hunk 4]
```diff
@@ -40,7 +40,7 @@
 			M.prop('environment', '{{{server.environment}}}');
 			M.prop('queryParams', {{{json queryParams}}});
 			M.prop('tracking', {{{json tracking}}});
-			M.prop('translations', {{{json (translations wikiVariables.language.content ns='main')}}});
+            M.prop('translations', {{{json (translations wikiVariables.language.content ns=localSettings.translationFiles)}}});
```
Comment (kvas-damian) [inline server/views/_layouts/ember-main.hbs]: Fix tabs vs spaces.

File: config/localSettings.base.ts [hunk 5]
```diff
@@ -116,7 +116,8 @@ var localSettings: LocalSettings = {
 			enable: true,
 			url: 'https://services.wikia.com/clickstream/events/social'
 		}
-	}
+	},
+	translationFiles: ['main','discussion']
```
Comment (kvas-damian) [inline config/localSettings.base.ts]: space after coma?

File: server/views/_helpers/translations.js (hunk 2 above)
Comment (Bloodmoon24) [inline server/views/_helpers/translations.js]: Good idea, I missed that.

File: config/localSettings.base.ts (hunk 5 above)
Comment (Bloodmoon24) [inline config/localSettings.base.ts]: Done

File: server/views/_layouts/ember-main.hbs (hunk 4 above)
Comment (Bloodmoon24) [inline server/views/_layouts/ember-main.hbs]: Done

File: server/views/_helpers/translations.js (hunk 3 above)
Comment (Bloodmoon24) [inline server/views/_helpers/translations.js]: I'll remember that

File: server/views/_helpers/translations.js (hunk 1 above)
Comment (Bloodmoon24) [inline server/views/_helpers/translations.js]: Done

File: front/templates/main/components/discussion-connection-error.hbs [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <div class="error-content">
 	{{i18n 'discussion.connection-error'}}
 	<a class="action-item" {{action attrs.retry}} >{{svg 'icon-retry' role='img' class='icon-retry'}}
-		<span>{{i18n 'discussion.action-retry-title'}}</span>
+		<span>{{i18n 'main.action-retry-title'}}</span>
```
Comment (kvas-damian) [inline front/templates/main/components/discussion-connection-error.hbs]: Don't we need `ns` here?

Comment (Bloodmoon24) [inline front/templates/main/components/discussion-connection-error.hbs]: Indeed, we need.

**Comment by kvas-damian** [comment]
LGTM

**Comment by bkoval** [comment]
LGTM :sandal:

**Pull request merged**